DOC_CHANGES.md

```diff
@@ -83,11 +83,13 @@ to better align with other ssh related options.
 
 `windows_package` now supports more than just `MSI`. Most common windows installer types are supported including Inno Setup, Nullsoft, Wise and InstallShield. The new allowed `installer_type` values are: `inno`, `nsis`, `wise`, `installshield`, `custom`, and `msi`.
 
+Non `:msi` package names should match the softwares `DisplayName` property expected to be found in the registry keys mentioned below. Further, they must explicitly include a package `source` attribute when using the `:install` action. It will not default to the package name if missing. Without the name matching the software display name, non `:msi` packages will always reconverge on `:install`.
```

Comment:
`DisplayName` is a registry key property, not a chef resource property. we should be more clear about that. It's not that we don't say that, but `DisplayName` sticks out (besides being camel case) to me as something I might set on my resource.

For proper behavior, the `package_name` property for Windows packages other than MSIs must be set to match ...

On second thought, we should remove/combine this paragraph with the one below that says almost the same thing, lists the registry key locations, but also mentions that the add/remove name might be more accessible.